packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -282,12 +282,17 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
     {
       entity_id: {
         type: "UUID",
-        primaryKey: true,
+        notNull: true,
+      },
+      version: {
+        type: "TIMESTAMP WITH TIME ZONE",
```

Comment:
Why are we storing a timezone? I'm pretty sure we just want to use UTC time-stamps